index.js.flow

```diff
@@ -0,0 +1,17 @@
+/* @flow strict */
+
+declare module '@github/image-crop-element' {
+  declare export default class ImageCropElement extends HTMLElement {
+    image: HTMLImageElement;
+    box: HTMLElement;
+    dragStartX: ?number;
+    dragStartY: ?number;
+    startX: number;
+    startY: number;
```

Comment:
These should be stored in a state map instead.

I am happy to work on this if this refactor is more than what you bargained for. 😬

Reply:
> I am happy to work on this if this refactor is more than what you bargained for. 😬

It's all good 😸